prov/usnic/src/usdf_domain.c

```diff
@@ -344,3 +344,15 @@ usdf_domain_open(struct fid_fabric *fabric, struct fi_info *info,
 	}
 	return ret;
 }
+
+char *
+usdf_domain_getname(uint32_t version, struct usd_device_attrs *dap)
+{
+	char *bufp;
+	if (FI_VERSION_GE(version, FI_VERSION(1, 4)))
+		bufp = strdup(dap->uda_devname);
+	else
+		bufp = strdup("usnic");
```

Comment:
Per https://github.com/ofiwg/libfabric/issues/2052, shouldn't `bufp` return NULL when `<= v1.3`?  (i.e., revert the behavior of ofiwg/libfabric@ca81e01823ffcfe26e1a18f5f85eaf57f1fba591)